src/solvers/dgsem_unstructured/sort_boundary_conditions.jl

```diff
@@ -67,6 +67,7 @@ function initialize!(boundary_types_container::UnstructuredSortedBoundaryTypes{N
     else
         for key in keys(boundary_dictionary)
             if !(key in unique_names)
+                @show unique_names
```

Comment:
Debug statement used while developing? Could you please remove it or improve the error message below?